unit-tests/func/frame-filter/test-l500-frame-filter.py

```diff
@@ -0,0 +1,60 @@
+import pyrealsense2 as rs, test
+import time
+
+# The L515 device opens the IR steam with the depth stream even if the user did not asked for it (for improving the depth quality), 
+# The frame-filter role is to  make sure only requested frames will get to the user.
+# This test is devided into 2 test.
+#   1. User ask for depth only - make sure he gets depth only frames
+#   2. User ask for depth + IR - make sure he gets depth + IR frames
+
+devices = test.find_devices_by_product_line_or_exit(rs.product_line.L500)
+device = devices[0]
+depth_sensor = device.first_depth_sensor()
+
+try:
+    dp = next(p for p in depth_sensor.profiles if p.stream_type() == rs.stream.depth)
+    irp = next(p for p in depth_sensor.profiles if p.stream_type() == rs.stream.infrared)
+except RuntimeError as e:
+    test.unexpected_exception()
+    test.finish()
+
+depth_frame_cnt = 0
```

Comment:
I don't like `cnt` -- either spell it out `count` or use `n_` prefix (`n_depth_frames`) which are more readable and an easier convention.

Reply:
👍